src/octoprint/filemanager/storage.py

```diff
@@ -453,6 +453,9 @@ def __init__(self, message, code=None, cause=None):
             code = StorageError.UNKNOWN
         self.code = code
 
+    def __str__(self):
+        return self.message
+
```

Comment:
I think instead of reimplementing `__str__` it makes more sense to adjust the constructor to something like this:

``` python
def __init__(self, message, code=None, cause=None):
  Exception.__init__(self, message)
  # ...
```

That way the inheritance is correct (`BaseException` here is actually a bug) and we reuse the default `__str__` handler of `Exception`.

Reply:
Looked if self.message is used outside this class but can't find anything, so dropped it, too.